include/deal.II/fe/fe.h

```diff
@@ -2991,6 +3008,72 @@ FiniteElement<dim,spacedim>::face_system_to_component_index (const unsigned int
 
 template <int dim, int spacedim>
 inline
+const FiniteElement<dim,spacedim> &
+FiniteElement<dim,spacedim>::get_child_element (const ComponentMask &mask) const
```

Comment:
I think this function might as well be in the .cc file. It's so long that inlining isn't likely going to benefit anything.